CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### backend-metal-0.6.4, backend-dx12-0.6.11, backend-dx11-0.6.9 (26-10-2020)
+  - specify entry point when shaders contain multiple entry points
```

Comment:
would be nice to tell what user impact this brings, e.g. "fixes a case where something something"

Reply:
:+1: it should be clearer now